Readme.md

```diff
@@ -3,20 +3,42 @@
-<!-- default file list end -->
-# How to resize ASPxSplitter placed inside ASPxPopupControl when resizing ASPxPopupControl
+# Popup Control for ASP.NET Web Forms - How to resize control content (Splitter) when resizing the popup
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3559/)**
 <!-- run online end -->
 
+This example demonstrates how to resize the nested [ASPxSplitter](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSplitter) control when the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) is resized.
+
+![Popup resizing](popup-resizing.gif)
 
-<p>At present, ASPxPopupControl does not have the built-in capability to resize nested controls when it's resizing. This example illustrates how to automatically resize controls places inside the popup. To accomplish this task, execute the following steps:</p><p>1. Set the <a href="http://documentation.devexpress.com/#AspNet/DevExpressWebASPxPopupControlASPxPopupControlBase_ScrollBarstopic"><u>ASPxPopupControl.ScrollBars</u></a> property to "Auto". This option will allow the popup control to evaluate the content width and height.</p><p>2. Handle the <a href="http://documentation.devexpress.com/#AspNet/DevExpressWebASPxPopupControlScriptsASPxClientPopupControlBase_AfterResizingtopic"><u>ASPxClientPopupControl.AfterResizing</u></a> event. In this event handler, call the <a href="http://documentation.devexpress.com/#AspNet/DevExpressWebASPxClassesScriptsASPxClientControl_AdjustControltopic"><u>AdjustControl</u></a> method for a desired control to force it to reevaluate its size.</p><p>3. Place the control into the <strong><div style="height: 100%; width: 100%; overflow: hidden"></strong> element. This element is necessary to overcome scrollbars that can be shown when ScrollBars="Auto" is set.</p>
+The [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) does not have the built-in capability to resize nested controls when it is resized. This example illustrates how to automatically resize controls places inside the popup. 
 
-<br/>
+To accomplish this task, execute the following steps:
 
+1. Set the [ASPxPopupControl.ScrollBars](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControlBase.ScrollBars) property to `Auto`. This option allows the popup control to evaluate the content width and height.
+    ```html
+    <dx:ASPxPopupControl ID="ASPxPopupControl1" runat="server" Height="500px" Width="500px" 
+                         ScrollBars="Auto" AllowResize="true" LoadContentViaCallback="OnFirstShow">
+        ...
+    </dx:ASPxPopupControl>
+    ```
+2. Handle the [ASPxClientPopupControl.AfterResizing](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.AfterResizing) event. In the event handler, call the [AdjustControl](https://docs.devexpress.com/AspNet/js-ASPxClientControl.AdjustControl) method for a nested control to force it to reevaluate its size.
```

Comment:
In the event handler, call the [AdjustControl](https://docs.devexpress.com/AspNet/js-ASPxClientControl.AdjustControl) method for a nested control to force it [it what: control or method?] to reevaluate its size.

Reply:
```suggestion
2. Handle the [ASPxClientPopupControl.AfterResizing](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.AfterResizing) event. In the event handler, call the [AdjustControl](https://docs.devexpress.com/AspNet/js-ASPxClientControl.AdjustControl) method for a nested control to force the control to reevaluate its size.
```